tests/test-basic-c.c

```diff
@@ -54,6 +55,25 @@ input_stream_to_bytes (GInputStream *input)
   return g_memory_output_stream_steal_as_bytes (G_MEMORY_OUTPUT_STREAM (mem_out_stream));
 }
 
+static void test_repo_exists (gconstpointer data)
+{
+  OstreeRepo *repo = OSTREE_REPO (data);
+  gboolean exists = 0;
+  GError *error = NULL;
+  /* Test a return value of TRUE on a repo that exists */
+  int success = ostree_repo_exists(repo, exists, error);
+  g_assert (success && exists);
+  /* Test a return value of FALSE on a repo that does not exist */
+  // Create an empty directory
+  mkdir ("not-a-repo",  S_IRUSR | S_IWUSR | S_IROTH | S_ IWOTH);
+  // Create a repo struct associated with that directory
+  GFile path = g_file_new_for_path ("not-a-repo");
```

Comment:
You need a `*` here.  I'd also use `g_autoptr()` to free it, so:

`g_autoptr(GFile) path = g_file_new...`